docs/setup-and-installing.md

```diff
@@ -2,151 +2,152 @@
+
+    |     |     |
+    | --- | --- |
+    | SSH | `git clone git@github.com:spatialos/UnityGDK.git` |
+    |HTTPS | `git clone https://github.com/spatialos/UnityGDK.git` | 
+    
 
     The GDK repository is a SpatialOS project called `UnityGDK`. It contains:
     - a Unity project at `UnityGDK/workers/unity`, which you need to open to use the GDK
     - SpatialOS features, such as the schema and snapshot files
     - development code
 
-    The Unity project contains the core GDK code, the `TransformSynchronization` Feature Module (more feature modules to come), and the `Playground`, which is an example scene using the features of the GDK.
-
-    The diagram below shows the structure of the `UnityGDK` SpatialOS project:
-
-	![UnityGDK structure](assets/UnityGDK-structure.png)
-
 ### Running a game from your Unity Editor
 
 You can now run a game from your Unity Editor, either using SpatialOS locally, or using SpatialOS in the cloud.
 
 Currently, you can try this out using the `Playground`.
 
-#### 2. Run the `Playground` locally using SpatialOS
+#### 1. Run the `Playground` locally using SpatialOS
 
 1. Open Unity, and open the `unity` project.
-<br>Unity will automatically download several required SpatialOS libraries.
+    - Unity will download several required SpatialOS libraries.
 Unity may open a browser window prompting you to login to your SpatialOS account.
 If this happens, please login.
 This will only happen the first time you open you project, or if the version of the required libraries changes.
 
-1. Choose **SpatialOS** > **Local launch**.
-<br>This launches a SpatialOS deployment locally in a new console window. You can open the [Inspector](https://docs.improbable.io/reference/13.0/shared/glossary#inspector) and see what’s happening in the game.
-    > **It’s done when:** You see `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector` printed in the new console indow
-
 1. In the Unity Editor's Project window, open **Assets** > **Playground** > **Scenes** > **SampleScene**.
 
+1. Choose **SpatialOS** > **Local launch**.
+    - This launches a SpatialOS deployment locally in a new console window. You can open the [Inspector](https://docs.improbable.io/reference/13.0/shared/glossary#inspector) and see what’s happening in the game.
+    > **It’s done when:** You see `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector` printed in the new console window
+
 1. Click the play icon.
-<br>You can move around using the `W`, `A`, `S`, and `D` keys, and move the camera using your mouse.
+    - You can move around using the `W`, `A`, `S`, and `D` keys, and move the camera using your mouse.
 
 1. When you want to stop the game:
     1. In the Editor, click the play icon again to stop your client.
     1. Stop the SpatialOS deployment by closing the console window.
 
-#### 3. Run the `Playground` in the cloud using SpatialOS
+#### 2. Run the `Playground` in the cloud using SpatialOS
```

Comment:
this is basically duplicating the build-and-deploy page. should we merge these two pages?

Reply:
If you want to grab these parts and put them in your PR - be my guest, just check with @ElleEss first :)

Reply:
currently doing it ;)